.github/workflows/build-mesa.yaml

```diff
@@ -0,0 +1,34 @@
+name: "Build OpenGL/Vulkan CTS"
+
+on:
+  workflow_dispatch:
+    inputs:
+      arch:
+        description: 'Runner to run the workflow'
+        required: true
+        default: 'ubuntu-24.04'
+        type: choice
+        options:
+          - ubuntu-24.04
+          - ubuntu-24.04-arm
+
+jobs:
+  run-install-mesa:
+    name: "Run install-mesa to generate CTS binaries"
+    runs-on: [ "${{ inputs.arch }}" ]
```

Comment:
maybe worth putting in the 'noble' flag as well? for me it didn't build on jammy iirc

Reply:
What do you mean by "We didn't enable the canonical local runner for the project."? Is it on purpose? I ran my job on a free canonical selfhosted openstack runner, is it something you want to avoid? ubuntu version of course can be configured in the input variables.